### ISSUE TITLE
 com.hazelcast.jet.core.JobRestartWithSnapshotTest.stressTest_suspendAndResume

### ISSUE THREAD

**Issue by cangencer** [hazelcast/hazelcast-jet#1111]
https://hazelcast-l337.ci.cloudbees.com/view/Jet/job/jet/com.hazelcast.jet$hazelcast-jet-core/931/testReport/junit/com.hazelcast.jet.core/JobRestartWithSnapshotTest/stressTest_suspendAndResume/

```
Error Message
java.lang.AssertionError: no snapshot found
Stacktrace
java.util.concurrent.ExecutionException: java.lang.AssertionError: no snapshot found
	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.util.concurrent.FutureTask.get(FutureTask.java:192)
	at com.hazelcast.jet.core.JobRestartWithSnapshotTest.stressTest(JobRestartWithSnapshotTest.java:456)
	at com.hazelcast.jet.core.JobRestartWithSnapshotTest.stressTest_suspendAndResume(JobRestartWithSnapshotTest.java:426)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:98)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.AssertionError: no snapshot found
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertNotNull(Assert.java:712)
	at com.hazelcast.jet.core.JobRestartWithSnapshotTest.waitForNextSnapshot(JobRestartWithSnapshotTest.java:364)
	at com.hazelcast.jet.core.JobRestartWithSnapshotTest.lambda$stressTest$17(JobRestartWithSnapshotTest.java:453)
	... 2 more
```

**Comment by viliam-durina** [comment]
I'd like to fix this one after #1104 is merged because it got changed there

**Comment by viliam-durina** [comment]
Lot of changes in that code since the failure, even the assert that failed was removed.
